scripts/mbedtls_framework/c_parsing_helper.py

```diff
@@ -129,3 +151,16 @@ def read_function_declarations(functions: Dict[str, FunctionInfo],
                                        return_type,
                                        name,
                                        arguments)
+
+_C_TYPEDEF_DECLARATION_RE = r'typedef (?:struct ){0,1}(?P<type>\w+) (?P<name>\w+)'
+
+def read_typedefs(filename: str)->  Dict[str, str]:
+    """ Extract type definitions in a {typedef name: primitiv type} dictionary"""
```

Comment:
Typo: primitiv → primitive. But that's not actually a primitive type, it's just whatever the type was defined as an alias of. Only for certain typedefs, given the regex. Can you please document which typedefs are caught here?